R/FilterStateChoices.R

```diff
@@ -355,7 +352,13 @@ ChoicesFilterState <- R6::R6Class( # nolint
           "are not in choices of column", private$varname, "in dataset", private$dataname, "."
         ))
       }
-      values[in_choices_mask]
+      values <- values[in_choices_mask]
+
+      if (length(values) != 1 && !private$multiple) {
+        warning("\"values\" is not a vector of length one. The first value will be selected.")
```

Comment:
Please fix this message to indicate which `dataname` and `varname` it refers to
```suggestion
        warning("\"values\" is not a vector of length one. The first value will be selected.")
```

Reply:
@kartikeyakirar do you mind to change the message to something more meaningful?